### PR TITLE
kubernetes behavior with image side loading and :latest tag

### LINKED ISSUE #328: document kubernetes behavior with image side loading and :latest tag
steps to reproduce:
1) build a container with tag: `latest`,
2) then load into kind, no error reported:
`kind load docker-image ###:latest`
3) then deploy pod to kind, check pod status, it getting image Pull Error.

it works well with other numeric tag as i tested.
thanks to @BenTheElder to point it out.

### PR DESCRIPTION
**Pull request by tao12345666333**
Resolves #328 

/kind documentation

Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

### CODE REVIEW

**Comment by tao12345666333** [comment]
https://deploy-preview-356--k8s-kind.netlify.com/docs/user/quick-start/#loading-an-image-into-your-cluster

File: site/content/docs/user/quick-start.md [hunk 1]
```diff
@@ -122,6 +122,19 @@ kind load docker-image my-custom-image:unique-tag
 kubectl apply -f my-manifest-using-my-image:unique-tag
 ```
 
+**Note**: The Kubernetes default pull policy is `IfNotPresent` which
```
Comment (BenTheElder) [inline site/content/docs/user/quick-start.md]:
```suggestion
**Note**: The Kubernetes default pull policy is `IfNotPresent` unless the image tag is `:latest` in which case the default policy is `Always`. `IfNotPresent`
```

**Comment by BenTheElder** [comment]
thanks! one suggestion

File: site/content/docs/user/quick-start.md (hunk 1 above)
Comment (tao12345666333) [reply site/content/docs/user/quick-start.md]: Thanks for your suggestion.

**Comment by tao12345666333** [comment]
Addressed. 

PS: I really want to try the `commit suggestion` function of GitHub. But it seems that there is no automatic squash commits function, I manually updated it. :laughing:

**Comment by BenTheElder** [comment]
> PS: I really want to try the `commit suggestion` function of GitHub. But it seems that there is no automatic squash commits function, I manually updated it. 😆

💯 , thanks. I think we sent them feedback about this once so far, but I totally agree. that feature is so nice except it makes a new commit for every suggestion accepted 😬

**Pull request merged**